docs/docs/Get-Started/get-started-quickstart.md

```diff
@@ -4,188 +4,502 @@ slug: /get-started-quickstart
+    previous_answer = result
+```
 
-1. In the **File** component, upload a text file from your local machine with data you want to ingest into the **Astra DB** component database.
-This example uploads an up-to-date CSV about Oscar winners.
-2. Click **Playground** to start a chat session.
-3. Ask the bot: `Who won the Oscar in 2024 for best movie?`
-4. The bot's response should be similar to this:
+  </TabItem>
+  <TabItem value="JavaScript" label="JavaScript">
+
+```js
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+const url = 'http://localhost:7860/api/v1/run/29deb764-af3f-4d7d-94a0-47491ed241d6';
+
+// Store the previous answer
+let previousAnswer = null;
+
+async function askAgent(question) {
+    const payload = {
+        "output_type": "chat",
+        "input_type": "chat",
+        "input_value": question
+    };
+
+    const options = {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(payload)
+    };
+
+    try {
+        const response = await fetch(url, options);
+        const data = await response.json();
+        
+        // Extract the message from the nested response
+        const message = data.outputs[0].outputs[0].outputs.message.message;
+        return message;
+    } catch (error) {
+        return `Error: ${error.message}`;
+    }
+}
+
+async function startChat() {
+    console.log("\nAsk the agent anything, such as 'What is 15 * 7?' or 'What is the capital of France?'");
+    console.log("Type 'quit' to exit or 'compare' to see the previous answer");
+
+    const askQuestion = () => {
+        rl.question('\nYour question: ', async (userQuestion) => {
+            if (userQuestion.toLowerCase() === 'quit') {
+                rl.close();
+                return;
+            }
+
+            if (userQuestion.toLowerCase() === 'compare') {
+                if (previousAnswer) {
+                    console.log(`\nPrevious answer was: ${previousAnswer}`);
+                } else {
+                    console.log("\nNo previous answer to compare with!");
+                }
+                askQuestion();
+                return;
+            }
+
+            const result = await askAgent(userQuestion);
+            console.log(`\nAgent's answer: ${result}`);
+            previousAnswer = result;
+            askQuestion();
+        });
+    };
+
+    askQuestion();
+}
+
+startChat(); 
+```
 
-```text
-The Oscar for Best Picture in 2024 was awarded to "Oppenheimer,"
-produced by Emma Thomas, Charles Roven, and Christopher Nolan.
+  </TabItem>
+</Tabs>
+
+## Apply temporary overrides to a flow run
+
+To change a flow's values at runtime, Langflow offers **Temporary Overrides**, also called **Tweaks**.
+Tweaks are added to the API request to the `/run` endpoint, and temporarily change component parameters within your flow.
```

Comment:
Are they only available for the /run endpoint?

You can combine this line with 475 which is basically saying the same thing.

Reply:
Poor wording, it should be "to the API request", then the next sentence should specify which endpoint. Fixed.